pfs_middleware/pfs_middleware/middleware.py

```diff
@@ -838,7 +842,9 @@ def __call__(self, req):
                     pop_and_restore(req.environ, 'swift.authorize_override',
                                     False):
                 if auth_cb and req.environ.get('swift.source') != 'PFS':
-                    req.acl = self._fetch_appropriate_acl(ctx)
+                    if not is_bypass_request:
+                        req.acl = self._fetch_appropriate_acl(ctx)
+                    # else, user needs to be swift owner
```

Comment:
Even if _we_ don't call `get_container_info`, middlewares left of us likely **will**. And likely in a fresh pre-auth'ed env, so no `X-Bypass-ProxyFS` header... hmm... :thinking: